### DISCUSSION TITLE
Prims and Display Layers

### DISCUSSION THREAD

**Discussion by jossh-h** [Autodesk/maya-usd#2669] [Q&A] [answered]
Hi there,

We are looking at switching over from the AL plugin to the maya usd plugin and were looking at options
to handle selectability in the viewport. We were managing this in the AL plugin with some custom prim metadata.

Switching to the maya usd plugin, the natural solution would be to use display layers and looking at the [docs](https://help.autodesk.com/view/MAYAUL/2023/ENU/index.html?guid=GUID-C52465E4-FA5F-46AF-A186-E20ABEDD9BF2) it looks to be compatible with UFE/USD sceneItems.

Though with our testing we cant seem to get this to work.. We are able to add prims to the display layer but toggling various options (Visibility, Reference, Template) don't have any effect in the viewport.
also running  `cmds.help('editDisplayLayerMembers')` does not show the [ufeObjects](https://github.com/Autodesk/maya-usd/blob/07103a39247b0108b73730f41ad7742a1d20d75f/test/lib/ufe/testDisplayLayer.py#L80) arg.. is there something we are missing to get this working?

**Comment by seando-adsk** [comment]
What version of Maya are you runing?

**Reply by jossh-h** [reply]
Hi @seando-adsk, we are testing on maya-2023.2

**Comment by seando-adsk** [comment, marked as answer]
Display Layers have only limited support in Maya 2023.2. They will need to be updated in a future version of Maya. We will update this discussion with details when we have them.

Sean

**Reply by jossh-h** [reply]
Ok thanks, we may have to re-think how we will handle this when we switch plugins.

Cheers,
Josh
